codespell_lib/data/dictionary.txt

```diff
@@ -41790,6 +41802,7 @@ wasnt'->wasn't
 wasnt->wasn't
 wasnt;->wasn't
 wass->was
+wast->waste
```

Comment:
Seen in https://github.com/Gnucash/gnucash/pull/1630/files#diff-3f07e4bd0a3bf3357de78daed5d95d17a4dd63336578c0440b3417b73cd13d4aR178

Reply:
```
wast->waste, waist,
```

Should it move to _rare_ because of "_thou [wast](https://dictionary.cambridge.org/dictionary/english/wast)_" like "_thou [doest](https://github.com/codespell-project/codespell/pull/2895/files)_"?